packages/components/src/internal/components/chart/ChartFieldOption.tsx

```diff
@@ -0,0 +1,230 @@
+                                        />
+                                    </div>
+                                    <div className="field-option-radio-group">
+                                        <label>Range</label>
+                                        <RadioGroupInput
+                                            name="scaleType"
+                                            options={SCALE_RANGE_TYPES.map(
+                                                option =>
+                                                    ({
+                                                        ...option,
+                                                        selected: scale.type === option.value,
+                                                    }) as RadioGroupOption
+                                            )}
+                                            onValueChange={onScaleTypeChange}
+                                            formsy={false}
+                                        />
+                                    </div>
+                                    {scale.type === 'manual' && (
+                                        <div className="chart-builder-scale-range-inputs">
+                                            <input
+                                                name="scaleMin"
+                                                type="number"
+                                                className="chart-builder-field-footer-input"
+                                                placeholder="Min"
+                                                onBlur={onScaleRangeBlur}
+                                                onChange={onScaleMinChange}
+                                                value={scale.min ?? ''}
+                                            />
+                                            <span>&nbsp;&nbsp;-&nbsp;</span>
+                                            <input
+                                                name="scaleMax"
+                                                type="number"
+                                                className="chart-builder-field-footer-input"
+                                                placeholder="Max"
+                                                onBlur={onScaleRangeBlur}
+                                                onChange={onScaleMaxChange}
+                                                value={scale.max ?? ''}
+                                            />
+                                            {invalidRange && (
+                                                <div className="text-danger">Invalid range (Max &lt;= Min)</div>
+                                            )}
+                                        </div>
+                                    )}
+                                </Popover>
+                            }
+                        >
+                            <i className="fa fa-gear" />
```

Comment:
nit: This should be `<span className="fa fa-gear" />`. The `<i>` tag is meant to format text, not render icons.

Reply:
done